test/plugs/assign_organization_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Community.AssignOrganizationTest do
+  use Community.ConnCase, async: true
+  alias Community.AssignOrganization
+  alias Community.Organization
+
+  describe "call" do
+    test "assigns the organization" do
+      conn = AssignOrganization.call(%Plug.Conn{}, [])
+
+      assert %Organization{} = conn.assigns[:organization]
```

Comment:
It's unclear from this test where the `%Organization{}` that is assigned here comes from. It's expected that a test has nothing in the db to start.